spec/requests/solidus_pay_tomorrow/create_order_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SolidusPayTomorrow::CreateOrderController, type: :request do
+  describe '#create' do
+    subject(:call) do
+      post '/api/order/pay_tomorrow', params: params
+    end
+
+    let(:payment_method) { create(:pt_payment_method) }
+    let(:user) { create(:user, email: 'user@bolt.com') }
+    let(:params) { { order_number: order.number, payment_method: payment_method.id, order_token: order.guest_token } }
+    let(:order) { create(:order) }
+
+    before do
+      allow(SolidusPayTomorrow::Client::CreateOrderService).to receive(:call).with(order: order,
+        payment_method: payment_method).and_return(
+          { url: "https://api.paytomorrow.com/verify/personal?app=order-token&auth=auth-token",
+            token: 'order-token' }.stringify_keys
+        )
+    end
+
+    context 'when order is created successfully' do
+      it 'returns created status code' do
+        call
+        expect(response).to have_http_status(:created)
+      end
+    end
+  end
```

Comment:
You added the rescue, so you should add a test for it.

Reply:
Will add

Reply:
Added in https://github.com/nebulab/solidus_pay_tomorrow/pull/9/commits/bd16050623ac9409f748fc49c53b2056f9636922